README.md

```diff
@@ -90,6 +90,12 @@ One of our goals is to make the transition from ARM Templates to Bicep as easy a
 
 Note that while we want to make it easy to transition to Bicep, we will continue to support and enhance the underlying ARM Template JSON language. As mentioned in [What is Bicep?](#what-is-bicep), ARM Template JSON remains the wire format that will be sent to Azure to carry out a deployment.
 
+**What unique benefits do you get with Bicep?**
```

Comment:
We have a section above the FAQ called "How is life better with bicep?" - can we combine this with that? I don't have a strong opinion on if it's here in the FAQ or in it's own section.

Reply:
That works